### PR TITLE
adding work description for next week

### PR DESCRIPTION
**Pull request by mhofsche**
contains a list of work items for next week

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # mhi
 git tests
 Mary new work Monday
+
+New desription for next weeks work:
+1. build support scafalding
```
Comment (brickhof) [inline README.md]: Please correct spelling: scaffolding

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+2. clean putz
+3. redo putz with armored mortor
```
Comment (brickhof) [inline README.md]: Please add information about where and when to get supplies

Comment (mhofsche) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (mhofsche) [reply README.md]: done

**Pull request merged**